app/connectors/MiddleConnector.scala

```diff
@@ -52,6 +52,9 @@ trait MiddleConnector {
   def connectToAts(UTR: SaUtr, taxYear: Int)(implicit hc: HeaderCarrier): Future[AtsData] =
     http.GET[AtsData](url("/taxs/" + UTR + "/" + taxYear + "/ats-data"))
 
+  def connectToPayeAts(nino: Nino, taxYear: Int)(implicit hc: HeaderCarrier): Future[AtsData] =
+    http.GET[AtsData](url("/taxs/" + nino.nino + "/" + taxYear + "/paye-ats-data"))
```

Comment:
Are there different types of Nino? I will have the check the interface spec. Thx.

Reply:
Apologies I wasn't clear - some systems are built to require the suffix (i.e., A, B, C or D), some require you to omit it, and some accept either. It would be sensible to clarify this for the endpoint you are using (although this is more a concern for the backend service - I just thought about it when looking here).